billing.go

```diff
@@ -52,6 +52,7 @@ type Billing struct {
 
 	// Amazon
 	AmazonAgreementID string `xml:"amazon_billing_agreement_id,omitempty"`
+	AmazonRegion      string `xml:"amazon_region,omitempty"`
```

Comment:
Do you know if the `amazon_region` field is available on `v2.15`? I looked through the latest version (`v2.17`) and `v2.15` but can't find that field in the documentation. I couldn't find it in the official PHP or Ruby clients recurly maintains either.

I'm sure the field exists, but since it's not documented I just want to confirm the field/type/spelling as well as the API version it appears on to make sure this is ready to merge into `master`.

Reply:
@cristiangraz The `amazon_region` field will be added in version `2.18`. I'd like to put this PR on hold until the release of `2.18` which is scheduled for January 29, 2019. I've updated the title accordingly.